### PR TITLE
Structure the `version` file

### PR DESCRIPTION
**Pull request by serhii-lekariev**

### CODE REVIEW

File: version.gradle.kts [hunk 1]
```diff
@@ -25,6 +25,12 @@
  * as we want to manage the versions in a single source.
  */
 
-val spineBaseVersion: String by extra("1.5.21")
-val spineTimeVersion: String by extra("1.5.21")
-val versionToPublish: String by extra("1.5.21")
+val versionToPublish = "1.5.22"
+val base = "1.5.22"
```
Comment (alexander-yevsyukov) [inline version.gradle.kts:29]: `base` what?

File: version.gradle.kts [hunk 2]
```diff
@@ -25,6 +25,12 @@
...
+val time = "1.5.22"
```
Comment (alexander-yevsyukov) [inline version.gradle.kts:30]: Why `time`?

Comment (alexander-yevsyukov) [reply version.gradle.kts:30]: Пол-второго и двадцать две секунды?

**Comment by serhii-lekariev** [comment]
See https://github.com/SpineEventEngine/core-java/pull/1283.

**Pull request closed without merging**